mysql-plugin/src/main/java/io/cdap/plugin/mysql/MysqlDBRecord.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.mysql;
+
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.plugin.db.ColumnType;
+import io.cdap.plugin.db.DBRecord;
+
+import java.util.List;
+
+/**
+ * Writable class for MySQL Source/Sink.
+ * TODO: Adding this class is part of a quick fix for CDAP-19417.
+ * The correct way to fix the jira is to use the top level input format to return StructuredRecord.
```

Comment:
@chtyim Please correct this comment

Reply:
The correct fix is :
1. in AbstractDBSource , instead of using DataDrivenETLDBInputFormat, we should use a new input format which directly implements InputFormat<LongWritable,  StructuredRecord> and delegates its functions to DataDrivenETLDBInputFormat
2. Each db specific plugin should export its own packages without export the dependent packages from database-common project

Reply:
two more thing
1. implement DB specific input format to extend the new input format ( so that we don't export the same new input format in every db specific plugin)
2. do similar for output format

Reply:
Meanwhile please add a jira for the TODO and put the Jira in the comments.

Reply:
Since we are fixing it. I think it is better to also make the effort to move the `exportcontent` in the pom.xml to individual children modules and have a more specific package to export.

Reply:
Fixed the export in this PR. Created a jira for the other part - https://cdap.atlassian.net/browse/PLUGIN-1346